src/django/api/serializers/activity_log.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    EmailField,
+    DateTimeField,
+)
+
+from ..models import User, Submission
+
+
+class ActivityDraftedSerializer(ModelSerializer):
+    user = EmailField(source='created_by.email')
+    action = SerializerMethodField()
+    time = DateTimeField(source='created_at')
+
+    def get_action(self, value):
+        return 'Drafted'
+
+    class Meta:
+        model = Submission
+        fields = ['user', 'action', 'time', 'notes']
```

Comment:
Drafts should not have notes:

![image](https://user-images.githubusercontent.com/1430060/196277746-6cc911b9-5cff-4631-8372-1c1aeb757619.png)